pom.xml

```diff
@@ -80,6 +93,7 @@
         <typesafe.config.version>1.3.0</typesafe.config.version>
         <lombok.version>1.16.20</lombok.version>
         <cleartk.version>2.0.0</cleartk.version>
+        <lucene-solr.version>7.4.0-SNAPSHOT</lucene-solr.version> <!-- TODO: change to 7.4.0 once it's released -->
```

Comment:
This should match the https://repository.apache.org/content/groups/snapshots/org/apache/solr/solr-solrj/7.4.0-SNAPSHOT/ content but the build fails, need to look into why, maybe a typo somewhere.